docs/build/reference/linker-property-pages.md

```diff
@@ -319,6 +327,10 @@ Specifies link-time code generation. ([`/LTCG`](ltcg-link-time-code-generation.m
 - **Profile Guided Optimization - Optimization** - Specifies that the linker should use the profile data created after running the instrumented binary to create an optimized image.
 - **Profile Guided Optimization - Update** - Allows and tracks list of input files to be added or modified from what was specified in the `:PGINSTRUMENT` phase.
 
+### Link Time Code Generation Object File
+
+Specifies *`.iobj`* file location. ([`/LTCGOUT:[name]`](ltcgout.md))
```

Comment:
Found a typo in the product itself:

<img width="1273" height="856" alt="Image" src="https://github.com/user-attachments/assets/278a7560-8b9d-47d9-8049-700d9814e8a6" />